lib/compute/jenkins-main-node.ts

```diff
@@ -112,13 +117,15 @@ export class JenkinsMainNode {
         assumedBy: new ServicePrincipal('ec2.amazonaws.com'),
       }),
       initOptions: {
-        timeout: Duration.minutes(20),
         ignoreFailures: props.failOnCloudInitError ?? true,
       },
       vpc: props.vpc,
       vpcSubnets: {
         subnetType: SubnetType.PRIVATE_WITH_NAT,
       },
+      minCapacity: 1,
```

Comment:
I assume what it does is if the main node goes down it will just bring up another one, only for that purpose with no intention to have multiple main nodes live together.